src/melee/ft/chara/ftMewtwo/ftMewtwo_SpecialHi.c

```diff
@@ -393,13 +402,12 @@ inline void ftMewtwo_SpecialHi_SetVars(HSD_GObj* fighter_gobj)
     func_80088148(fp, 0x30DA1U, SFX_VOLUME_MAX, SFX_PAN_MID);
 }
 
-extern BOOL func_8009A134(HSD_GObj*);
-
 // 0x80145990
 // https://decomp.me/scratch/kFSF2 // Mewtwo's grounded Teleport Zoom Action
 // State handler
 void ftMewtwo_SpecialHi_Action(HSD_GObj* fighter_gobj)
 {
+    /// @todo Shared @c inline with #ftMewtwo_SpecialAirHi_Action.
```

Comment:
There are SO MANY of these. I gave up on marking them at some point. And they're really hard to detect automatically, I think.